cypress/support/step-definitions/simple-color-picker-steps.js

```diff
@@ -1,23 +1,11 @@
-import {
-  simpleColorPickerDiv,
-  simpleColorPickerLegend,
-} from "../../locators/simple-color-picker";
+import { simpleColorPickerLegend } from "../../locators/simple-color-picker";
 import {
   experimentalSimpleColorPickerInput,
   simpleColorPicker,
 } from "../../locators/advanced-color-picker/index";
 import { commonDataElementInputPreview } from "../../locators";
 import { keyCode } from "../helper";
 
-Then("Simple Color Picker {int} element was picked up", (index) => {
```

Comment:
Was this unused?

Reply:
it came back some how after I've already removed once